About.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kamulete training center</title>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet" />
+        <script src="https://kit.fontawesome.com/ecb44d8fff.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+    <header>
+        <nav class="twitter">
+            <a href="#">Twitter</a>
+            <a href="#">Whatsap</a>
+            <a href="#">training</a>
+            <a href="#">My page</a>
+            <a href="#">logout</a>
+        </nav>
+        <nav class="navlogo">
+            <a class="img-logo" href="index.html"><img src="img-folder/logo-image---00.png" alt=""></a>
+            <div class="hamburg">
+                <div class="hamburg1"></div>
+                <div class="hamburg2"></div>
+                <div class="hamburg3"></div>
+            </div>
```

Comment:
- [ ] On your mobile design, the about page is missing all the navigation links as a result there is no link back to the homepage so kindly fix this as well :100:
- Your Design Now
![image](https://user-images.githubusercontent.com/102699958/225422710-f8a547d8-4fd0-4c40-b3aa-ffadbd857123.png)